chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/PendingCertifiedProductManager.java

```diff
@@ -215,6 +217,9 @@ public void deletePendingCertifiedProduct(final Long pendingProductId)
     @Transactional
     @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).PENDING_CERTIFIED_PRODUCT, "
             + "T(gov.healthit.chpl.permissions.domains.PendingCertifiedProductDomainPermissions).CONFIRM, #acbId)")
+    @CacheEvict(value = {
+            CacheNames.DEVELOPER_NAMES
```

Comment:
Sort of a duplicate comment but I would recommend moving this to the createFromPending method in another class.